### PR TITLE
Chore: ident files and remove some useless code

### PR DESCRIPTION
**Pull request by kyazdani42**
This PR is not that useful but at least code is indented properly.
I also removed some code in `parsers.lua` that wasn't needed/used anymore.

### CODE REVIEW

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -17,16 +17,12 @@ local list = setmetatable({}, {
 
     rawset(table, parsername, setmetatable(parserconfig, {
       __newindex = function(parserconfigtable, key, value)
-        if key == "used_by" then
-          ft_to_parsername[value] = parsername
```
Comment (theHamsta) [inline lua/nvim-treesitter/parsers.lua:21]: why is `used_by` not needed anymore?

File: lua/nvim-treesitter/parsers.lua [hunk 2]
```diff
@@ -454,20 +450,6 @@ function M.ft_to_lang(ft)
   return ft_to_parsername[ft] or ft
 end
 
-function M.lang_to_ft(lang)
```
Comment (theHamsta) [inline lua/nvim-treesitter/parsers.lua:457]: this code is used

File: lua/nvim-treesitter/parsers.lua (hunk 1 above)
Comment (kyazdani42) [reply lua/nvim-treesitter/parsers.lua:21]: i believe used_by is already set after the rawset call, removing this code did not affect highlighting in jsx files so i believed it was ok to remove it.

Comment (theHamsta) [reply lua/nvim-treesitter/parsers.lua:21]: It is set in the update_ft_to_parser name that is called two lines after the removed code.

**Review by theHamsta — APPROVED** [review]

**Pull request merged**